### PR TITLE
data:Abhinav-6

### PR DESCRIPTION
**Pull request by Abhinav-6**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4601"></a>

### CODE REVIEW

**Review by rahulpandey70 — CHANGES_REQUESTED** [review]
Hey, @Abhinav-6 please rename your json file name to data/Abhinav-6.json.

**Comment by Abhinav-6** [comment]
What's wrong with this. I have done everything according to the guide. Am I missing something.

**Comment by rahulpandey70** [comment]
> What's wrong with this. I have done everything according to the guide. Am I missing something.

Your file name should match to your github username.
like this :- ```data/Abhinav6.json to data/Abhinav-6.json```

File: data/Abhinav6.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Abhinav Kumar",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Frontend developer | diving deep into **Node.js** | **TypeScript** Enthusiast",
+  "avatar": "https://github.com/Abhinav-6.png",
```
Comment (swagataroy30) [inline data/Abhinav6.json:6]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

**Review by maharshi-sinha — CHANGES_REQUESTED** [review]
There's no need of avatar! It'll automatically pick up your github profile pic.

**Review by Hrishi5111998 — CHANGES_REQUESTED** [review]
hey @Abhinav-6 please remove comma.

**Comment by Abhinav-6** [comment]
I've edited the filename and remooved the avatar field. Now i think it is ready to merge.

**Review by priyankarpal — DISMISSED** [review]
LGTM 👍🏻

File: data/Abhinav-6.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "C++",
+    "Node.js"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/_Abhinav6" },
+    { "icon": "FaGithub", "url": "https://github.com/Abhinav-6" },
```
Comment (Panquesito7) [inline data/Abhinav-6.json:17]: Extra comma in here (if I'm not wrong).
```suggestion
    { "icon": "FaGithub", "url": "https://github.com/Abhinav-6" }
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by priyankarpal — APPROVED** [review]
Awesome 🎉 let's get this merge.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**